cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -798,16 +803,22 @@ impl ExprOrSpecial<'_> {
     }
 
     fn into_entity_type(self) -> Result<ast::EntityType> {
-        self.into_name().map(ast::EntityType::from)
+        self.into_unreserved_name().map(ast::EntityType::from)
     }
 
-    fn into_name(self) -> Result<ast::Name> {
+    fn into_unreserved_name(self) -> Result<ast::Name> {
```

Comment:
should we go back to calling this `into_name()`?

Reply:
Done.